cmdstanpy/model.py

```diff
@@ -1579,6 +1596,12 @@ def _run_cmdstan(
                 env=os.environ,
                 universal_newlines=True,
             )
+            if timeout:
+                timer = threading.Timer(timeout, proc.terminate)
```

Comment:
Should we later be calling `timer.cancel()` if the task finishes before this timeout hits?

Reply:
Ah neat, didn't know about that function.